controllers/secretscope_controller_databricks.go

```diff
@@ -152,25 +152,66 @@ func (r *SecretScopeReconciler) submitACLs(instance *databricksv1alpha1.SecretSc
 	return nil
 }
 
-func (r *SecretScopeReconciler) submit(instance *databricksv1alpha1.SecretScope) error {
+// checkCluster checks if Databricks cluster supports ACLs, and checks if secret scope exists.
+func (r *SecretScopeReconciler) checkCluster(instance *databricksv1alpha1.SecretScope) error {
```

Comment:
Could you please rename "checkCluster" to "VerifyACL"?
You can create a secret scope without having a cluster and ACL is not related to the cluster. Using the access control list (ACL) is only available with the Azure Databricks Premium Plan.
I don't know much about ACL, we can have a chat with @lace about it and get her opinion.
https://docs.azuredatabricks.net/security/secrets/secret-acl.html

Reply:
I've renamed it from `checkCluster` to `verifyWorkspace`. Felt it was more fitting because we're checking the workspace for secrets and ACL support.